usaspending_api/transactions/delta_models/transaction_search.py

```diff
@@ -1066,11 +1070,22 @@
                     )
                 ),
                 TRUE
-            ) AS tas_components
+            ) AS tas_components,
+        CASE
+            WHEN SIZE(COLLECT_SET(rpa.program_activity_name)) > 0
+                THEN COLLECT_SET(rpa.program_activity_name)
+            ELSE NULL
+        END AS program_activity_names,
+        CASE
+            WHEN SIZE(COLLECT_SET(rpa.program_activity_code)) > 0
+                THEN COLLECT_SET(rpa.program_activity_code)
+            ELSE NULL
+        END AS program_activity_codes
         FROM int.financial_accounts_by_awards AS faba
         INNER JOIN global_temp.treasury_appropriation_account AS taa ON taa.treasury_account_identifier = faba.treasury_account_id
         INNER JOIN global_temp.federal_account AS fa ON fa.id = taa.federal_account_id
         INNER JOIN global_temp.toptier_agency agency ON (fa.parent_toptier_agency_id = agency.toptier_agency_id)
+        LEFT JOIN global_temp.ref_program_activity rpa ON (faba.program_activity_id = rpa.id)
```

Comment:
Since you are doing a `LEFT JOIN` onto a series of tables that are using `INNER JOIN` can you confirm that `INNER JOIN` of `faba`, `taa`, `fa`, and `agency` will not omit any program activity?

Reply:
I don't remember if this is guaranteed by business logic or not